src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js

```diff
@@ -4,24 +4,27 @@ import type {ElementRef} from 'react';
 import classNames from 'classnames';
 import Icon from '../Icon';
 import Loader from '../Loader';
-import type {FieldTypeProps} from '../../types';
 import inputStyles from './input.scss';
 
 const LOADER_SIZE = 20;
 
-type Props = FieldTypeProps<?string> & {
+type Props = {
     name?: string,
     icon?: string,
     type: string,
     loading?: boolean,
     placeholder?: string,
     inputRef?: (ref: ?ElementRef<'label'>) => void,
-    onFocus?: () => void,
+    valid: boolean,
+    value: ?string,
+    onBlur?: () => void,
+    onChange: (value?: string) => void,
```

Comment:
That already had some quite strange errors earlier... I think we should write `value: ?string` instead.